docs/open-payments.md

```diff
@@ -0,0 +1,7 @@
+# Open Payments
+
+Open Payments is an API standard that allows third-parties (with the account holder's consent) to initiate payments and to view the transaction history on the account holder's account. Extensive documentation can be found on the [Open Payments website](https://openpayments.guide).
```

Comment:
ChatGPT said:

"Open Payments is an API standard that lets third-parties, with account holder consent, initiate payments, view transaction history, and more."